src/design/content-design/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Content design at dxw
```

Comment:
This should have `related_order` and `last_reviewed_at`. Same on a few other pages